ui/src/AddRecipe/AddRecipe.js

```diff
@@ -28,6 +28,7 @@ class AddRecipe extends Component {
       instructions: [],
       name: "",
       time: "",
+      link: "",
```

Comment:
we probably should call that imageURL so it shows up in the database properly

Reply:
Changed the stuff from link to imageURL